data/human/hails.txt

```diff
@@ -747,6 +747,8 @@ phrase "friendly civilian"
 		"uninstall any radar jammers on your ship. They're useless"
 		"use loans to get money to make money"
 		"reduce the cost of your ships by using batteries over bigger generators"
+		"jump into hostile systems at a large distance from the center"
```

Comment:
You can't do this intentionally(?)

Reply:
That is true. I originally wrote this as a reference to #4432 for an Author ship, but this text might be confusing for players when they see a merchant saying this, so I'll remove this text here.